### PR TITLE
update coverimage

### PR DESCRIPTION
**Pull request by TimoStahl**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -218,14 +218,14 @@
     "coverimage": {
         "title": "Coverimage",
         "version": "1.0.38.1",
```
Comment (creecros) [inline plugins.json]: "version": "1.2.5",

This refers to the plugin version, so what you have in the Plugin.php is what this needs to be.

File: plugins.json [hunk 2]
```diff
@@ -218,14 +218,14 @@
...
         "license": "MIT",
-        "description": "This plugin adds a coverimage function to the board.",
+        "description": "This plugin adds a coverimage function to tasks on the board, and allows you to set a project image to use as a logo for projects.",
         "homepage": "https://github.com/BlueTeck/kanboard_plugin_coverimage",
         "readme": "https://raw.githubusercontent.com/BlueTeck/kanboard_plugin_coverimage/master/README.md",
-        "download": "https://github.com/BlueTeck/kanboard_plugin_coverimage/releases/download/1.0.38.1/Coverimage.zip",
+        "download": "https://github.com/BlueTeck/kanboard_plugin_coverimage/releases/download/1.2.5.0/Coverimage.zip",
         "remote_install": true,
-        "compatible_version": ">=1.0.38"
+        "compatible_version": ">=1.2.5"
```
Comment (creecros) [inline plugins.json]: "compatible_version": ">=1.0.42" 

Tested and works. This refers to the Kanboard version that the plugin is compatible with. it is compatible with 1.0.38, but the project overview was different then, so the project logo won't work until 1.0.42, when the dashboard was improved.

**Review by creecros — CHANGES_REQUESTED** [review]
Few edits

**Comment by TimoStahl** [comment]
thanks, changed

**Pull request merged**